chef/cookbooks/ceph/providers/pool.rb

```diff
@@ -24,8 +24,8 @@ def get_pg_num(ceph_conf, admin_keyring)
 end
 
 def create_pool(ceph_conf, admin_keyring, pool_name, pg_num)
-  Chef::Log.info("Creating ceph pool '#{pool_name}' with pg number '#{pg_num}'")
-  cmd = "ceph -k #{admin_keyring} -c #{ceph_conf} osd pool create '#{pool_name}' '#{pg_num}'"
+  Chef::Log.info("Creating ceph pool #{pool_name} with pg number #{pg_num}")
+  cmd = "ceph -k #{admin_keyring} -c #{ceph_conf} osd pool create #{pool_name} #{pg_num}"
```

Comment:
@mjura as I said on IRC, if you remove the quotes, then you need full escaping. If you keep the quotes, then you need to escape quotes inside the variable